Lombiq.Gulp.Extensions.csproj

```diff
@@ -15,7 +15,9 @@
   
   <Target Name="InitCommonNodeModules" BeforeTargets="ResolveReferences" Inputs="package.json" Outputs="$(InitCommonNodeModulesStampFile)">
     <Exec Command="npm install" />
-    <Exec Command="move node_modules ../.." Condition="Exists('node_modules')" />
+    <Exec Command="move node_modules ../.." Condition="'$(OS)' == 'Windows_NT' And Exists('node_modules')" />
+    <Exec Command="mv node_modules ../.." Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' And Exists('node_modules')" />
+    <Exec Command="mv node_modules ../.." Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' And Exists('node_modules')" />
```

Comment:
Did you try the Move task? https://docs.microsoft.com/en-us/visualstudio/msbuild/move-task That might work as well in a single command, without having to run different commands for each OS.

If that doesn't work then please move the OSX and Linux ones into one command, the two OS checks OR-d together.